### PR TITLE
fix python layer in fft module

### PR DESCRIPTION
**Pull request by KsanaKozlova**

### CODE REVIEW

**Review by densmirn — COMMENTED** [review]
Let's create variable with value `0` to see what it means. E.g. `norm`.

File: dpnp/fft/dpnp_iface_fft.py [hunk 1]
```diff
@@ -86,10 +86,12 @@ def fft(x1, n=None, axis=-1, norm=None):
...
     if x1_desc:
-        # if norm is None or norm is 'backward':
-        #     norm_val = 0
-        # else:
-        #     norm_val = 1
+        norm_val = 0
+        if norm is "forward":
+            norm_val = 1
+        elif norm is "ortho":
+            norm_val = 2
```
Comment (densmirn) [inline dpnp/fft/dpnp_iface_fft.py:93]: I would propose to create `enum` for `norm`. What do you think about that?

File: dpnp/fft/dpnp_iface_fft.py [hunk 2]
```diff
@@ -69,6 +70,22 @@
 ]
 
 
+class Norm(Enum):
+    backward = 0
+    forward = 1
+    ortho = 2
+
+def _norm_value(norm):
```
Comment (densmirn) [inline dpnp/fft/dpnp_iface_fft.py:78]: Maybe rename it to `get_validated_norm`?

File: dpnp/fft/dpnp_iface_fft.py [hunk 3]
```diff
@@ -69,6 +70,22 @@
...
+    if norm is None or norm == "backward":
```
[the remark this answers is not in the document]
Comment (densmirn) [reply dpnp/fft/dpnp_iface_fft.py:79]: My bad, let's keep the check.

**Review by densmirn — APPROVED** [review]

**Review by Alexander-Makaryev — APPROVED** [review]
LGTM

**Pull request merged**